crates/store/re_sorbet/src/column_schema.rs

```diff
@@ -35,6 +35,16 @@ pub enum ColumnDescriptor {
 }
 
 impl ColumnDescriptor {
+    /// Debug-only sanity check.
```

Comment:
There's nothing in this method that guarantees debug-only behavior though?

Reply:
No guarantees other than the docstring contract